cmd/run_konan

```diff
@@ -85,13 +85,14 @@ KOTLIN_SCRIPT_RUNTIME_JAR="${KONAN_HOME}/konan/lib/kotlin-script-runtime.jar"
 STUB_GENERATOR_JAR="${KONAN_HOME}/konan/lib/StubGenerator.jar"
 INTEROP_INDEXER_JAR="${KONAN_HOME}/konan/lib/Indexer.jar"
 INTEROP_JAR="${KONAN_HOME}/konan/lib/Runtime.jar"
-SHARED_JAR="${KONAN_HOME}/konan/lib/shared.jar"
+SHARED_JAR="${KONAN_HOME}/konan/lib/kotlin-native-shared.jar"
```

Comment:
Would be great to combine all JARs we can into the single one eventually, to avoid changes here.